### PR TITLE
Audio.js - set isPlaying to false in stop()

### PR DESCRIPTION
**Pull request by smatthews1999**
Audio.js - set isPlaying to false in stop()

### CODE REVIEW

File: src/audio/Audio.js [hunk 1]
```diff
@@ -116,7 +116,8 @@ THREE.Audio.prototype = Object.assign( Object.create( THREE.Object3D.prototype )
 			return;
 
 		}
-
+		
+		this.autoplay = false;
```
Comment (mrdoob) [inline src/audio/Audio.js]: Don't you mean `isPlaying`?

**Comment by smatthews1999** [comment]
Yes of course... sorry again.  

Let me try again.

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
Added it to `pause()` too 😊

**Comment by smatthews1999** [comment]
Ok will do.

**Comment by mrdoob** [comment]
No no, I meant that I already added it.
